src/redux/features/booksSlice.js

```diff
@@ -0,0 +1,23 @@
+import { createSlice } from "@reduxjs/toolkit";
+const initialState = []
+
```

Comment:
![image](https://github.com/DjibrilM/book-store/assets/93039359/3da0cbc6-b06f-438b-b127-7c98351644be)

- [ ] It would be nice if you remove the features folder and make separate folders for the **booksSlice.js** and **categoriesSlice.js** files to follow the Ducks pattern :+1: as suggested by the previous reviewer :blush: